iroha-balancer-client/src/main/kotlin/soramitsu/irohautils/balancer/client/service/IrohaBalancerClientService.kt

```diff
@@ -99,3 +95,18 @@ open class IrohaBalancerClientService @JvmOverloads constructor(
         const val LIST_TORII_QUEUE_NAME = "list-torii"
     }
 }
+
+class ClientErrorException : Exception {
```

Comment:
`Exception` contains `message`, just reuse it using constructor as you actually do, why do you need those custom fields?

Reply:
I took it from Bakong and Unkai. I'll update that